test/Model.js

```diff
@@ -319,6 +325,7 @@ describe('Model', function (){
       model.should.have.property('id', 1);
       model.should.have.property('name', 'Fluffy');
       model.should.have.property('vet', { address: '12 somewhere', name: 'theVet' });
+      model.should.have.property('profileImage', imageData);
```

Comment:
This whole section should probably be included in a separate test, not this one.